data/panel.js

```diff
@@ -8,7 +8,13 @@ countEventLoopLags.addEventListener("click", function() {
   self.port.emit("mode-changed", "eventLoopLags");
 });
 var hangThreshold = document.getElementById("hangThreshold");
-hangThreshold.addEventListener("input", function() {
```

Comment:
I think I actually preferred the form, since we'd get the enter key-press handling for free.

If you switch this event listener from "input" to "change", I think you'll get what you want - it won't fire until the field has been changed _and_ blurred.